### DISCUSSION TITLE
pg_dump: error: aborting because of server version mismatch

### DISCUSSION THREAD

**Discussion by MrGeneralZod** [docker-library/postgres#1161] [General]
I updated the last vesion postgis (16-master). After that I tried to back up my data using pgAdmin. But backup process was failed.
There was a error message "pg_dump: detail: server version: 16.0 (Debian 16.0-1.pgdg110+1); pg_dump version: 15.3"

![image](https://github.com/docker-library/postgres/assets/15141643/5c28dda1-ca83-4213-859d-5abe65d2f302)

**Comment by ImreSamu** [comment]
I see that the pgAdmin4 docker image doesn't yet support the new PG16. 
Someone has already reported this issue in that repository.
* https://github.com/pgadmin-org/pgadmin4/issues/6768

**Comment by akshay-joshi** [comment]
You will have to provide the Binary Path in the Preferences dialog for PG 16.

<img width="1046" alt="Screenshot 2023-09-21 at 11 30 52 AM" src="https://github.com/pgadmin-org/pgadmin4/assets/1632882/8728144e-a7c6-4237-8e27-6c50c7223db6">

**Reply by thelanceprogramming** [reply]
this works!
